app/controllers/api/greetings_controller.rb

```diff
@@ -0,0 +1,6 @@
+class Api::GreetingsController < ApplicationController
+  def random
+    random_greeting = Message.order('RANDOM()').first
+    render json: { message: random_greeting.content }
+  end
+end
```

Comment:
- [ ] I can't see any route that points to this controller, please consider adding it! 🤓 🔧 

- [ ] Also, you didn't run any migrations or create the Message model. Please consider using them! 🤓 🔧

Reply:
Resolved